packages/mdc-dialog/foundation.js

```diff
@@ -82,6 +82,7 @@ export default class MDCDialogFoundation extends MDCFoundation {
   }
 
   open() {
+    this.isOpen_ = true;
```

Comment:
Pretty much; I'm a fan of setting internal state as the first thing in methods. This way you can use the updated internal state in the rest of the method body.